src/OpenTelemetry/Metrics/MeterProviderBuilderBase.cs

```diff
@@ -105,6 +109,22 @@ internal MeterProviderBuilder AddView(Func<Instrument, MetricStreamConfiguration
             return this;
         }
 
+        internal MeterProviderBuilder SetMaxMetricStreams(int metricStreamLimit)
```

Comment:
Is there a reason to use different names other than `SetMetricStreamsLimit(int limit)` (keeping the method name and the parameter name aligned)?

Reply:
no. just typo. will fix.